src/adaptive_graph_of_thoughts/tests/test_metadata_helpers.py

```
@@ -0,0 +1,473 @@
+from pathlib import Path
+from datetime import datetime, timezone
+import sys
+from typing import Dict, Any, List
+
+# Add the src directory to the path for importing the module under test
+sys.path.insert(0, os.path.join(os.path.dirname(__file__), '..', '..', '..'))
+
+from adaptive_graph_of_thoughts.metadata_helpers import (
+    MetadataManager,
+    ValidationError,
+    MetadataValidator,
```

Ruff (0.11.9)

`adaptive_graph_of_thoughts.metadata_helpers.MetadataValidator` imported but unused

Remove unused import: `adaptive_graph_of_thoughts.metadata_helpers.MetadataValidator`

(F401)